torch_geometric/utils/trim_to_layer.py

```diff
@@ -14,6 +14,19 @@
 )
 
 
+def filter_empty_tensor(input_tensor: Union[MaybeHeteroNodeTensor,
+                                            MaybeHeteroEdgeTensor]):
+    r"""Filter out empty tensor for x, edge_index, edge_attr after trim.
+    This can avoid some unnecessary computation when some node/edge
+    types get empty output"""
+    if input_tensor is None:
+        return
```

Comment:
```suggestion
        return None
```
Return `None` explicitly.